tasks/task963_librispeech_asr_next_word_prediction.json

```diff
@@ -0,0 +1,937 @@
+{
+    "Source": [
+        "librispeech_asr"
```

Comment:
Could you include other pointers to the resource? (URL, paper, etc)

Reply:
Yes, I'll add the URL for the dataset.